src/components/button.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {ButtonStyled, ButtonText} from './styles/button';
+import {FieldContainer} from './styles/general';
+
+interface buttonProps {
+  content: string;
+  pressButon: () => void;
+}
+
+export const StyledButton: React.FC<buttonProps> = props => {
```

Comment:
```suggestion
interface StyledButtonProps {
  content: string;
  pressButon: () => void;
}

export const StyledButton: React.FC<buttonProps> = props => {
```

Só pra já ir seguindo a convenção que temos no nome de interfaces de props.